Rlog/View/Components/UnderlinedTextField.swift

```diff
@@ -75,6 +77,48 @@ struct UnderlinedTextField<T>: View {
 }
 
 private extension UnderlinedTextField {
+    @ViewBuilder
+    var textFieldView: some View {
+        switch textFieldType {
+        case .workplace, .reason, .none:
+            underlinedTextFieldView
+        case .payday, .wage, .time:
+            underlinedNumberFieldView
+        }
+    }
+
+    var underlinedNumberFieldView: some View {
+        VStack {
+            HStack {
+                TextField(
+                    textFieldType.placeholder,
+                    value: $text,
+                    formatter: NumberFormatter()
+                )
+                .padding(.horizontal)
+                .keyboardType(textFieldType.keyboardType)
+                .focused($isNumberFieldFocused)
+
+                Spacer()
+
+                if textFieldType == .workplace {
+                    HStack {
+                        Spacer()
+                        Text("\(String(describing: text).count)/20")
+                            .font(.caption)
+                            .foregroundColor(.gray)
+                    }
```

Comment:
HStack을 한번 더 감싼 이유가 있을까요?
상위에 스페이서가 있기 때문에 스페이서가 없어도 괜찮을것같습니다.

```suggestion
                        Text("\(String(describing: text).count)/20")
                            .font(.caption)
                            .foregroundColor(.gray)
```

Reply:
cc. @phainestha1 
-> 테오와 직접 소통한 결과 추가 수정 예정이라고 합니다.